Documentation/Testing/ExtensionTesting.rst

```diff
@@ -30,30 +37,12 @@ Scope
 About this chapter and what it does *not* cover, first.
 
 * This documentation assumes an extension is tested with only one major Core version. It
-  does not support extension testing with multiple target Core versions. Extensions that
-  support multiple Core versions at the same time in the same branch are *not* scope of this document.
+  does not support extension testing with multiple target Core versions (though that is
```

Comment:
can we link to some ressource, which explains how to do that or does none exist yet?

Reply:
No, as far as I know we do not have a resource for that. The "Testing Extensions" is the only resource I know which fully explains this in the official documentation. What I would suggest to do is split up the "Testing Extensions" later into another page / section which covers the more advanced parts. We might also add something to "tea" documentation. 

For now I would like to update the already existing documentation, ideally before adding more.